src/parser.rs

```diff
@@ -6320,21 +6320,22 @@ impl<'a> Parser<'a> {
     }
 
     fn check_on_update_expr_is_valid(expr: &Expr) -> Result<(), ParserError> {
-        let valid_object_names: [ObjectName; 4] = [
-            ObjectName(vec!["CURRENT_TIMESTAMP".into()]),
-            ObjectName(vec!["LOCALTIME".into()]),
-            ObjectName(vec!["LOCALTIMESTAMP".into()]),
-            ObjectName(vec!["NOW".into()]),
+        const VALID_FN_NAMES: [&'static str; 4] = [
+            keywords::CURRENT_TIMESTAMP,
+            keywords::LOCALTIME,
+            keywords::LOCALTIMESTAMP,
+            "NOW",
         ];
 
         if let Expr::Function(f) = expr {
-            if valid_object_names.contains(&f.name) {
+            if VALID_FN_NAMES.contains(&f.name.to_string().as_str()) {
```

Comment:
I think this will now allocate each time to create a `String`

However, that does seem better than 4 allocations :)

Reply:
Yeah I figured 1 was better than four ... but didn't think to use the `Ident` in the `ObjectName` as there may be multiple .. however for this case we are only testing for functions that must only have one `Ident` for the `ObjectName`, other cases would be invalid anyway.

Updated the refactor commit and force pushed.